### PR TITLE
Java security jvm ttl

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

File: tools/jvm.java.security [hunk 1]
```diff
@@ -0,0 +1,2 @@
+networkaddress.cache.ttl=60
```
Comment (eshryane) [inline tools/jvm.java.security:1]: Use an application-specific name for the file like `whois.java.security` so it's clear it's separate from the JVM

File: whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java [hunk 2]
```diff
@@ -102,4 +104,18 @@ private void stopService(final ApplicationService applicationService, boolean fo
...
     }
+
+    private static void printJvmSecurityProperties() {
+        if(!Boolean.valueOf(Security.getProperty("security.overridePropertiesFile"))) {
+            LOGGER.warn("security.overridePropertiesFile is false, cannot override security values");
+        }
+
+        final String networdkAddrCacheTtl = Security.getProperty("networkaddress.cache.ttl");
+        LOGGER.info("networkaddress.cache.ttl: {}", networdkAddrCacheTtl);
+        LOGGER.info("networkaddress.cache.negative.ttl: {}", Security.getProperty("networkaddress.cache.negative.ttl"));
```
Comment (eshryane) [inline whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java:115]: Also create a variable for the negative TTL value and log separately

File: whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java [hunk 3]
```diff
@@ -102,4 +104,18 @@ private void stopService(final ApplicationService applicationService, boolean fo
...
+
+        if(networdkAddrCacheTtl == null || networdkAddrCacheTtl.equals("-1")) {
+            LOGGER.warn("Networkaddress.cache.ttl is not sert properly");
```
Comment (eshryane) [inline whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java:118]: Typo: "sert" -> "set"

File: whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java (hunk 2 above)
Comment (maggarwal13) [reply whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java:115]: I am logging the value for negative ttl, there is no need to create a seperate variable for it as its default is 10s  and I am not logging it as a warning

Comment (eshryane) [reply whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java:115]: The default may change (it's JVM specific), I think both should be logged in the same way

Comment (maggarwal13) [reply whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java:115]: ok, I will put warning for negative one too

**Review by eshryane — APPROVED** [review]

**Pull request merged**